tests/legacy-cli/e2e/tests/misc/module-resolution.ts

```diff
@@ -73,14 +73,14 @@ export default async function () {
 
   await updateJsonFile('tsconfig.json', tsconfig => {
     tsconfig.compilerOptions.paths = {
-      '@firebase/polyfill': ['@firebase/polyfill/index.ts'],
```

Comment:
What's going on with these paths? It looks like breaking backward compatibility.

Reply:
Bad test.  They were failing due to a bug (which this PR fixes) rather then the intended reason.  it should be failing due to non-project typescript files being pulled into the compilation which it now does.  Before it was failing due to missing files.